### DISCUSSION TITLE
Google Health Connect & Apple Health (more) support

### DISCUSSION THREAD

**Discussion by whirlpool-m51** [zepp-health/zeppos-forum#114] [Developer Feedback]
Hi Guys,

Seems like google put more efforts to develop [Google Health Connect API](https://developer.android.com/guide/health-and-fitness/health-connect) rather than Google Fitness REST API. When I look at some smart watch apps, they already moved (or moving) to Google Health Connect (e.g. Fitbit, Samsung galaxy, etc)

Just wonder, are you guys have a plan(or already developing) to support (or move) the Google Health Connect?

Also, I know you guys already support Apple Health but some data (such as the activities[excercises?]) is not syncing at this moment.
Do you have a plan to sync those activities data to Apple Health?

Lastly, is there a way to get some Zepp Health data using REST API (or GraphQL, etc)? Seem like fetch api is not for this case

**Comment by MyoungXUE** [comment]
Hi, we have plans to access and improve the data for both Google Health and Apple Health, and I understand that they will go live in the near future.

There are no plans to access Zepp Health's health data via API for now.
